en/docs/insights/derive-insights.md

```diff
@@ -0,0 +1,364 @@
+# Derive Insights
+
+This section explains how you can derive insights about the APIs deployed in your organization. 
+
+## Access the Insights Portal
+
+To access the Insights Portal, access the Choreo Console at [https://console.choreo.dev/](https://console.choreo.dev/), and click **Insights** in the left navigation menu.
+
+## Analyzing statistics
+
+Once you access the Insights Portal, you can view the following information.
+
+### Overview
+
+This page gives you a quick overview of the whole system. The targeted audiences for the page are managers and business users who need quick insight. Also, this page can be used as a dashboard to view the current system status.
+
+![Overview](../assets/img/insights/overview.png){.cInlineImage-full}
+
+The information displayed is as follows:
+
+- **Total Traffic**
+
+    ![Total traffic](../assets/img/insights/overview-page-total-traffic.png){.cInlineImage-half}
+
+    This widget displays the total traffic your selected environment received during the selected time range. This includes both successful requests and error requests. To investigate further, you can click the arrow icon on the bottom right corner of the widget to open the [**Traffic** page](#traffic)
+    
+- **Error request count**
```

Comment:
Seems like we have used title capitalization to reflect the widget name in the UI with regard to all other widgets. Should. we use title caps here too?

Reply:
Done